schemas/core/components/travel-mode.json

```diff
@@ -22,7 +22,7 @@
     "personalMode": {
       "description": "A mode that involves using your personal vehicle or legs",
       "type": "string",
-      "enum": ["BICYCLE", "CAR", "WALK"]
+      "enum": ["BICYCLE", "CAR", "WALK", "SHARED_BIKE"]
```

Comment:
For consistency it should be `SHARED_BICYCLE`.

Also I think we should introduce `meta.MODE_SHARED_BICYCLE` (same way as we did with `SHARED_CAR`)

Reply:
Hi @medikoo I addressed your remarks.